github/Requester.py

```diff
@@ -381,6 +381,15 @@ def _get_installation_authorization(self):
             permissions=self.__app_auth.token_permissions,
         )
 
+    def get_app(self):
```

Comment:
I think this method should go into `GithubIntegration`.

Reply:
I'd rather say GithubIntegration requires some rework to get the authentication, token refresh and Requester init args sorted out: https://github.com/PyGithub/PyGithub/pull/2461